src/assessments/seo/internalLinksAssessment.js

```diff
@@ -64,18 +45,9 @@ class TextHasInternalLinksAssessment extends Assessment {
 		this.linkStatistics = researcher.getResearch( "getLinkStatistics" );
 		let assessmentResult = new AssessmentResult();
 
-		if ( ! isEmpty( this.linkStatistics ) ) {
```

Comment:
Any reason for removing this extra safeguard? It might look as if it's not necessarily required, but there might also have been a reason for putting it there (some kind of bug that might otherwise occur under certain circumstances). I'd be very cautious with this, and in general I'd avoid removing extra safeguards which have previously been implemented.